boot/bootdata/livecd.inf

```diff
@@ -16,6 +16,9 @@ HKLM,"SYSTEM\CurrentControlSet\Services\Cdrom","Start",0x00010001,0x00000000
 ; Reset BootExecute to an empty value: AutoChk should not start in MiniNT mode
 HKLM,"SYSTEM\CurrentControlSet\Control\Session Manager","BootExecute",0x00010000,""
 
+; ComputerName
```

Comment:
```suggestion
; Host Name
```

Reply:
@binarymaster The registry key and value names are "computer name", so why should it be changed to "host" name?

Reply:
Because otherwise it's captain obvious-level comment, not explaining the meaning of key.